src/WorkflowManager/Contracts/Models/PayloadDto.cs

```diff
@@ -0,0 +1,45 @@
+﻿/*
+ * Copyright 2022 MONAI Consortium
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using Newtonsoft.Json;
+
+namespace Monai.Deploy.WorkflowManager.Contracts.Models
+{
+    public class PayloadDto : Payload
+    {
+        public PayloadDto() { }
+        public PayloadDto(Payload payload)
+        {
+            Version = payload.Version;
+            Id = payload.Id;
+            PayloadId = payload.PayloadId;
+            Workflows = payload.Workflows;
+            WorkflowInstanceIds = payload.WorkflowInstanceIds;
+            FileCount = payload.FileCount;
+            CorrelationId = payload.CorrelationId;
+            Bucket = payload.Bucket;
+            CallingAeTitle = payload.CallingAeTitle;
+            CalledAeTitle = payload.CalledAeTitle;
+            Timestamp = payload.Timestamp;
+            Files = payload.Files;
+            PatientDetails = payload.PatientDetails;
+            PayloadDeleted = payload.PayloadDeleted;
+        }
+
+        [JsonProperty(PropertyName = "payload_status")]
+        public string PayloadStatus { get; set; } = "";
```

Comment:
Should this be of type enum instead of any string?

Reply:
I would have probably had it as an enum yeah, I'll see how it's used and change it if it makes sense